cardinal/search_test.go

```diff
@@ -305,3 +305,197 @@ func TestSearchExample(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Equal(t, amount, 5)
 }
+
+func TestExactFilterOnSearch(t *testing.T) {
```

Comment:
```suggestion
func TestSearch_Exact_ReturnsExactComponentMatch(t *testing.T) {
```

The idea of this convention is `Test<relevant package/type/function>_<method or context>_<tested behavior>`

Reply:
There are cases where you have to bend the convention here, but the idea is to highlight the relevant function/methods/package and the behaviour that you want to pinpoint in the test.